docs/products/postgresql/howto/migrate-db-to-aiven-via-console.rst

```diff
@@ -0,0 +1,308 @@
+
+.. seealso::
+
+    For the other migration method (using ``aiven-db-migrate``), see :doc:`Migrate to Aiven for PostgreSQL® with aiven-db-migrate </docs/products/postgresql/howto/migrate-aiven-db-migrate>`.
+
+About migrating via console
+---------------------------
+
+* Console migration tool enables you to migrate PostgreSQL databases to clusters in your Aiven account. You can migrate
+
+  * Existing on-premise PostgreSQL databases
+  * Cloud-hosted PostgreSQL databases
+  * Managed database clusters on Aiven.
+
+* Console migration tool uses `logical replication <https://www.postgresql.org/docs/current/logical-replication.html>`_ by default.
+
+* With the console migration tool, you can migrate PostgreSQL® databases either using the continuous migration method, which is the one recommended and detailed in this guide, or using the dump method (``pg_dump``).
+
+  * Continuous migration keeps the source database operational during the migration. It requires either superuser permissions or the ``aiven_extras`` extension.
+
+    * Superuser permissions can be verified using the guidelines in :ref:`Check out how to verify that you have superuser permissions <check-superuser>`. If you do not have superuser permissions, you can request them from your system administrator.
+    * ``aiven_extras`` extension (`GitHub repository <https://github.com/aiven/aiven-extras>`_) can be installed on your source database using the guidelines in :doc:`Install or update an extension </docs/products/postgresql/howto/manage-extensions>`.
+
+  * Dump is a point-in-time snapshot and requires the source database to be offline or in the maintenance mode during the migration process since any data it gets after initiating the dump are not migrated to the target database.
+
+.. _check-superuser:
+
+.. dropdown:: Expand to check out how to verify that you have superuser permissions.
+
+    Use ``psql`` to run the ``\du`` command:
+
+    .. code-block:: bash
+
+       \du
+
+    .. code-block:: bash
+       :caption: Expected output
+
+       Role name |                      Attributes                            |                 Member of
+       ----------+------------------------------------------------------------+-----------------------------------------
+       _source_pg_db     | Superuser, Replication                                     | {}
+       example   | Create role, Create DB, Replication, Bypass RLS            | {pg_read_all_stats,pg_stat_scan_tables,pg_signal_backend}
+       postgres  | Superuser, Create role, Create DB, Replication, Bypass RLS | {}
+
+    Identify your role name in the ``Role name`` column and check if it has the ``Superuser`` attribute assigned in the ``Attributes`` column.
+
+.. topic:: No superuser permissions? Migrate using the dump method.
```

Comment:
We mentioned it's possible to use replication without superuser, but with `aiven_extras` extension installed on the source databases, so we should probable describe how. And only if that is also not possible for any reason - fallback to dump.